### PR TITLE
feat: add sorted set put and fetch by score

### PR DESCRIPTION
**Pull request by nand4011**
Add sorted set put element and put elements. Put elements can take in the sorted set in a variety of formats.

Add sorted set fetch by score. It returns its result as an array of hashes [{value: "value", score: 1.0}].

Fix some rubocop warnings

### CODE REVIEW

**Comment by nand4011** [comment]
I ignored the rubocop parameter list warning for sorted set fetch by score, since almost all the parameters are optional. We could evaluate a JS style options object if that makes more sense.

File: lib/momento/response/sorted_set/sorted_set_fetch_response.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    # The sorted set does not exist.
+    # @return [Boolean]
+    def miss?
+      false
+    end
+
+    # The fetched values as UTF-8 Strings and their scores.
+    # @return [(Array[{ value: String, score: Float }] | nil)] the UTF-8 elements and their scores
+    def value_string_elements
```
Comment (cprice404) [inline lib/momento/response/sorted_set/sorted_set_fetch_response.rb:22]: can we add an alias that is just `value` that defaults to returning `value_string_elements`?

**Review by cprice404 — APPROVED** [review]
LGTM. One suggestion about adding a shorthand `value` accessor, not a blocker.

File: lib/momento/response/sorted_set/sorted_set_fetch_response.rb (hunk 1 above)
Comment (nand4011) [reply lib/momento/response/sorted_set/sorted_set_fetch_response.rb:22]: Yeah. I'll add it in a follow on pr.

**Pull request merged**